cobra/core/solution.py

```diff
@@ -169,6 +169,11 @@ def y(self):
         warn("use solution.reduced_costs.values() instead", DeprecationWarning)
         return self.reduced_costs.values
 
+    def to_frame(self):
+        """Return the fluxes and reduced costs as a data frame"""
+        return DataFrame({'fluxes': self.fluxes,
```

Comment:
Is the index of the `DataFrame` automatically taken to be the index of the fluxes and reduced costs, i.e., the reaction IDs? If not I would add that explicitly.